addons/io_hubs_addon/components/definitions/media_frame.py

```diff
@@ -57,7 +57,8 @@ class MediaFrame(HubsComponent):
         'panel_type': [PanelType.OBJECT, PanelType.BONE],
         'icon': 'OBJECT_DATA',
         'deps': ['networked'],
-        'version': (1, 0, 0)
+        'version': (1, 0, 0),
+        'tooltip': 'Add media like images, PDFs, and videos'
```

Comment:
I'm fine with whatever wording you like, but would it be more clear to use something like 'Add a standard place for users to display media'?